Marlin/src/inc/Conditionals_post.h

```diff
@@ -1447,7 +1447,7 @@
 
 // Calculate a default maximum stepper rate, if not supplied
 #ifndef MAXIMUM_STEPPER_RATE
-  #define MAXIMUM_STEPPER_RATE (1000000UL / (2UL * (MINIMUM_STEPPER_PULSE)))
+  #define MAXIMUM_STEPPER_RATE (1000000UL / (2UL * ((MINIMUM_STEPPER_PULSE == 0) ? 1 : MINIMUM_STEPPER_PULSE)))
```

Comment:
Note that `MINIMUM_STEPPER_PULSE == 0` won't work if `MINIMUM_STEPPER_PULSE` isn't defined. (This isn't an `#if` statement.)

Reply:
@thinkyhead : What do you suggest to protect against MINIMUM_STEPPER_PULSE not defined ? --- I´d either force define it to 2 (will work for most cases), or force a compilation failure.

But i have no idea on the proper solution...

Reply:
For this kind of case we either supply a default that sets it to 0 when undefined, or we add a check everywhere it's used.

Reply:
setting to 0 would be fine, failing the compilation would also be fine.
All the example configurations are defining it to 2 or more

Your choice 👍  ... I can do the changes, or if you like, you can do them. I have no problem 👍